### ISSUE TITLE
[url] 'function' object has no attribute 'priority'

### ISSUE THREAD

**Issue by dgw** [sopel-irc/sopel#1173]
Sopel v. 6.5.0 is not loading the `url` module properly. I am unable to get titles using `.title http://url.here`, and reloading the module gives this error:

````
<dgw> Sopel: reload url
<Sopel> AttributeError: 'function' object has no attribute 'priority' (file
        "/usr/local/lib/python2.7/dist-packages/sopel/bot.py", line 213, in unregister)
````

Perhaps that's a red herring, because on startup Sopel prints to stdout: `Error in url setup procedure: nothing to repeat (../../../../../lib/python2.7/re.py:242)`

My config includes no `exclude` rules, and `exclusion_char = ^`. This occurs both with an empty `exclude` option in the config file and with no `exclude` line at all.

**Comment by dgw** [comment]
NB: Still present in 6.5.1.

**Comment by dgw** [comment]
OK, whatever happens, my proposed fix for #1032 also appears to fix this reloading issue.
